share/dictionary/radius/dictionary.f5

```diff
@@ -26,6 +26,13 @@ ATTRIBUTE	LTM-User-Info-1				12	string
 ATTRIBUTE	LTM-User-Info-2				13	string
 ATTRIBUTE	LTM-Audit-Msg				14	string
 
+ATTRIBUTE	F5-F5OS-UID					21	integer
```

Comment:
For v4, the attributes don't have a prefix which is the vendor name.  I'll pull the changes in manually.